src/views/Pools/components/CardFooter.tsx

```diff
@@ -94,6 +109,10 @@ const CardFooter: React.FC<Props> = ({
   const blocksUntilStart = Math.max(startBlock - block, 0)
   const blocksRemaining = Math.max(endBlock - block, 0)
 
+  const isMetamask = (window as WindowChain).ethereum && (window as WindowChain).ethereum.isMetaMask && tokenAddress
+
+  const imageSrc = `https://pancakeswap.finance/images/tokens/${tokenName.toLowerCase()}.png`
```

Comment:
Does it work if you don't specify the domain ? i.e `/images/tokens/${tokenName.toLowerCase()}.png`. Since it's passed to metamask, I'm not sure. 
If not put the base url in `/config/index.ts`, let's avoid to have hardcoded content like this, in case we change the url one day.

Reply:
Added to `/config/index.ts`